backend/docs/README.md

```
@@ -422,10 +422,32 @@ Generated By: create-express-doc
+```bash
+curl -X POST https://your-domain.com/api/clean-supabase/trigger \
+  -H "x-cron-secret: your-secret-here"
+```
 
 **Responses:**
 
+- **401** - Unauthorized
+  - Content-Type: `application/json`
+  - Schema:
+
+        ```json
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)